spec/javascripts/controllers/middleware/middleware_topology_controller_spec.js

```diff
@@ -18,23 +18,61 @@ describe('middlewareTopologyController', function () {
       "name": "Local",
       "kind": "MiddlewareServer",
       "miq_id": 1,
-      "status": "Unknown",
+      "status": "Running",
       "display_kind": "MiddlewareServer",
       "icon": "vendor-wildfly"
     }
   };
 
+  var mw_server_starting = {
+    id: "MiddlewareServer1", item: {
+      "status": "Starting"
+    }
+  };
+
+  var mw_server_reload = {
+    id: "MiddlewareServer1", item: {
+      "status": "Reload required"
+    }
+  };
+
+  var mw_server_down = {
+    id: "MiddlewareServer1", item: {
+      "status": "Down"
+    }
+  };
+
+  var mw_server_unknown = {
+    id: "MiddlewareServer1", item: {
+      "status": "Unknown"
+    }
+  };
+
   var mw_deployment = {
     id: "MiddlewareDeployment1",
     item: {
       "name": "hawkular-command-gateway-war.war",
       "kind": "MiddlewareDeployment",
       "miq_id": 1,
-      "status": "Unknown",
+      "status": "Enabled",
       "display_kind": "MiddlewareDeploymentWar"
     }
   };
 
+  var mw_deployment_disabled = {
+    id: "MiddlewareDeployment1",
```

Comment:
it's a nit, but I would set different ids to different vars in the test. from a consistency perspective only.